server_ui/advertise.go

```diff
@@ -101,19 +101,22 @@ func advertiseInternal(ctx context.Context, server server_utils.XRootDServer) er
 	}
 	serverUrl := param.Origin_Url.GetString()
 	webUrl := param.Server_ExternalWebUrl.GetString()
-	prefix := param.Origin_NamespacePrefix.GetString()
+	serverIssuer, err := config.GetServerIssuerURL()
```

Comment:
Just leaving this comment for the sake of posterity since we figured this out in person -- since the advertisement is for a server, we use the server's issuer URL. The function that generates this in `config` first checks for a value from the config file `Server.IssuerUrl` and then falls back to the server's external web url in the event that nothing is configured.